### PR TITLE
Make sure wsagent runs on openjdk11

### PR DESCRIPTION
**Pull request by eivantsov**
Adding a custom setenv.sh and bind+activation dependencies.

### CODE REVIEW

File: assembly/assembly-wsagent-server/src/assembly/setenv.sh [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+                                             -Dcom.sun.management.jmxremote.authenticate=false \
+                                             -Dche.local.conf.dir=${CHE_LOCAL_CONF_DIR}"
+
+#Class path
+[ -z "${CLASSPATH}" ]  && CLASSPATH="${CATALINA_HOME}/conf/:${JAVA_HOME}/lib/tools.jar"
+
+
+# On java9 runtime, enable activation and JAXB API
+#JAVA_VERSION=$(java -version 2>&1 | head -n 1 | cut -d'"' -f2 | cut -d'.' -f1 | sed 's/[^0-9]*//g')
+#[ "$JAVA_VERSION" -ge 9 ] && JAVA_OPTS="$JAVA_OPTS --add-modules java.activation --add-modules java.xml.bind"
```
Comment (nickboldt) [inline assembly/assembly-wsagent-server/src/assembly/setenv.sh]: we could pull this from upstream and just delete lines 37-39 (or comment out line 39) with an ant regex replacement.

File: assembly/assembly-wsagent-war/pom.xml [hunk 2]
```diff
@@ -24,6 +24,26 @@
     <packaging>war</packaging>
     <name>CodeReady Workspaces :: Workspace Agent</name>
     <dependencies>
+        <dependency>
+            <groupId>javax.xml.bind</groupId>
+            <artifactId>jaxb-api</artifactId>
+            <version>2.2.11</version>
```
Comment (nickboldt) [inline assembly/assembly-wsagent-war/pom.xml]: If this is inheriting from <dependencyManagement> we don't need versions here.

Comment (eivantsov) [reply assembly/assembly-wsagent-war/pom.xml]: Good catch. Fixed.

**Comment by nickboldt** [comment]
merged

**Pull request closed without merging**